content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -153,8 +153,10 @@ different Kubernetes components.
 | `Sysctls` | `true` | Beta | 1.11 | |
 | `TokenRequest` | `false` | Alpha | 1.10 | 1.11 |
 | `TokenRequest` | `true` | Beta | 1.12 | |
+| `TokenRequest` | `true` | GA | 1.20 | |
 | `TokenRequestProjection` | `false` | Alpha | 1.11 | 1.11 |
 | `TokenRequestProjection` | `true` | Beta | 1.12 | |
+| `TokenRequestProjection` | `true` | GA | 1.20 | |
```

Comment:
Please move these feature gates into the next section; they no longer belong under the heading _Feature gates for Alpha or Beta features_.

Reply:
done